### PR TITLE
Update SetFloatingHelpTextStyle.md

### PR DESCRIPTION
**Pull request by TomStilson**
Update detailed information about this native. Tested on local server.

### CODE REVIEW

File: HUD/SetFloatingHelpTextStyle.md [hunk 1]
```diff
@@ -6,14 +6,26 @@ aliases: ["0x788E7FD431BD67F1"]
...
-* **hudIndex**: 
-* **p1**: 
-* **p2**: 
-* **p3**: 
-* **p4**: 
-* **p5**: 
+* **hudIndex**: Hud index for the help text. `0` uses `FLOATING_HELP_TEXT_1` hud component, `1` uses `FLOATING_HELP_TEXT_2` hud component. See [`SHOW_HUD_COMPONENT_THIS_FRAME`](#_0x0B4DF1FA60C0E664) for index information.
+* **showArrow**: Whether or not to show a small arrow below the help text message box.
+* **colorIndex**: Background color for the message box and arrow(if enabled). See (HUD colors)[https://docs.fivem.net/docs/game-references/hud-colors/] for more info.
+* **p3**: Currently unknown. Set to -1 or 0 or any other integers does not seem to have effects.
```
Comment (4mmonium) [inline HUD/SetFloatingHelpTextStyle.md:17]: I'm somewhat sure about `p3` being opacity, ranging from `0x00` to `0xFF`, `0xFF` being fully opaque of course.

https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4

Here's a game function that calls a method this parameter is using:
```c
void __fastcall sub_14016DFB4(int a1)
{
  float v1; // xmm0_4
  float v2; // xmm2_4
  _DWORD *v3; // rax
  int v4[86]; // [rsp+20h] [rbp-158h] BYREF
  DWORD v5; // [rsp+180h] [rbp+8h] BYREF

  if ( a1 )
  {
    v1 = 0.050000001;
    if ( a1 <= 0 )
      v2 = -0.050000001;
    else
      v2 = 0.050000001;
    if ( (float)(*(float *)&dword_141CD61D0 + v2) >= 0.050000001 )
    {
      v1 = *(float *)&GRAPHICS_11_MISC_4;
      if ( (float)(*(float *)&dword_141CD61D0 + v2) <= 1.0 )
        v1 = *(float *)&dword_141CD61D0 + v2;
    }
    dword_141CD61D0 = LODWORD(v1);
    dword_141CD61D4 = *getPlayerColorLookup(&v5, dword_141CD61CC, (unsigned int)(int)(float)(v1 * 255.0));
    v5 = joaat_switch(0, "OPACITY");
    v3 = au_re_memmove_13(v4, &dword_141CD61B0);
    sub_1401D2204(0i64, v3, &v5);
  }
}
```

Comment (TomStilson) [reply HUD/SetFloatingHelpTextStyle.md:17]: > I am too lazy to verify this, but the parameters can be directly mapped to `SET_HELP_TEXT_STYLE` parameters in `hud.gfx` (although they are reordered by the native):
> 
> ```c
> int floatingHudIndex; // Native adds 1 so 0 corresponds to `HELP_TEXT_1`
> int currentStyle; // Reasonable values: -1, 0, 1; -1 disables all styling
> int hudColorIndex; // GET_HUD_COLOUR, etc.
> int hudColorAlpha; // Unused by scaleform
> int arrowPosition; // Requires currentStyle != 0|-1
> int boxOffset; // Requires arrowPosition [1, 4]
> ```

I'm not sure if there is p6 for this native, but I will change the parameter name. Thanks alot!

Comment (4mmonium) [reply HUD/SetFloatingHelpTextStyle.md]: Looks like the PR failed to validate:
```
ParseError: A parameter definition was found for a parameter that does not exist. Parameter name: floatingHudIndex
```
`floatingHudIndex` does not match the function argument (`hudIndex`) defined up above at (https://github.com/citizenfx/natives/pull/770/files#diff-b5e913fe1c09a7c16a57ff3aae3aab478d30723a9be036bc04f81bfd29a927c8R9) they both need to be named identically for it to verify :-)

Comment (TomStilson) [reply HUD/SetFloatingHelpTextStyle.md]: > This still fails to validate, make sure your [function arguments](https://github.com/citizenfx/natives/pull/770/files#diff-b5e913fe1c09a7c16a57ff3aae3aab478d30723a9be036bc04f81bfd29a927c8R9), match the function argument parameters displayed down [below](https://github.com/citizenfx/natives/pull/770/files#diff-b5e913fe1c09a7c16a57ff3aae3aab478d30723a9be036bc04f81bfd29a927c8R13).
> 
> ```
> ParseError: A parameter definition was found for a parameter that does not exist. Parameter name: currentStyle
> ```
> 
> Here's the [contribution guide](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md). Thanks!

I have fixed the documentation issue and now it should be good, thanks for the help!

Comment (4mmonium) [reply HUD/SetFloatingHelpTextStyle.md]: @TomStilson 
Seems good, please [squash](https://www.git-tower.com/learn/git/faq/git-squash) your commits whenever you can, thanks!

**Comment by 4mmonium** [comment]
Closing this since it has been superseded by PR #856, thank you for your effort nonetheless! 😊

**Pull request closed without merging**